### PR TITLE
Code update for Create custom Aggregation

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | 4.5+
| Edition       | N/A

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by papcio122 — APPROVED** [review]

**Review by adamwojs — COMMENTED** [review]
@MagdalenaZuba Have you tried the steps provided on the edited documentation page on your machine?

**Comment by MagdalenaZuba** [comment]
@adamwojs no, as I don't have Solr configured. That's why I asked the reporting developer to confirm whether the steps work.

**Comment by adamwojs** [comment]
Then please install Solr and Elasticsearch on your local environment, follow the steps from the article and paste the results (as a comment to this PR) of the example aggregation 

```php
use App\Query\Aggregation\PriorityRangeAggregation;
use Ibexa\Contracts\Core\Repository\Values\Content\Query\Aggregation\Range;
// ...
$aggregation = new PriorityRangeAggregation('example', [
    Range::ofInt(0, 100),
]);
``` 

attached to both Content & Location query on clean `ibexa/commerce` data. If you stump upon any issue I'm here to help.

**Comment by adriendupuis** [comment]
> attached to both Content & Location query on clean `ibexa/commerce` data. If you stump upon any issue I'm here to help.
@adamwojs On 4.5.3, the actual example is working only with Location as `priority_id` field is dedicated to Locations. Content items use `main_location_priority_i`.

**Comment by adamwojs** [comment]
> @adamwojs On 4.5.3, the actual example is working only with Location as priority_id field is dedicated to Locations. Content items use main_location_priority_i.

Point for you @papcio122. @MagdalenaZuba Could you please adapt the article?

File: docs/search/extensibility/create_custom_aggregation.md [hunk 1]
```diff
@@ -62,15 +62,15 @@ The example below uses `RangeAggregationVisitor`:
 === "Solr"
 
     ``` yaml
-    App\Query\Aggregation\Solr\PriorityAggregationVisitor:
+    App\Query\Aggregation\Solr\PriorityRangeAggregationVisitor:
```
Comment (papcio122) [inline docs/search/extensibility/create_custom_aggregation.md:65]: I would change it to be consistent with elasticsearch tab
```suggestion
    app.search.solr.query.aggregation_visitor.priority_range_aggregation:
```

File: docs/search/extensibility/create_custom_aggregation.md [hunk 2]
```diff
@@ -106,15 +106,16 @@ and provide it with the aggregation class in the `aggregationClass` parameter.
 
 === "Solr"
 
-    Tag the service with `ibexa.solr.query.location.aggregation_result_extractor`.
+    Tag the service with `ibexa.search.solr.query.location.aggregation.result.extractor`.
 
     ``` yaml
     App\Query\Aggregation\Solr\PriorityAggregationResultExtractor:
```
Comment (papcio122) [inline docs/search/extensibility/create_custom_aggregation.md:112]: I would change it to be consistent with elasticsearch tab
```suggestion
    app.search.solr.query.aggregation_result_extractor.priority_range_aggregation:
```

File: docs/search/extensibility/create_custom_aggregation.md [hunk 3]
```diff
@@ -106,15 +106,16 @@ and provide it with the aggregation class in the `aggregationClass` parameter.
...
         class: Ibexa\Solr\ResultExtractor\AggregationResultExtractor\RangeAggregationResultExtractor
         arguments:
             $aggregationClass: 'App\Query\Aggregation\PriorityRangeAggregation'
```
Comment (papcio122) [inline docs/search/extensibility/create_custom_aggregation.md:115]: The service is missing an argument
```suggestion
            $aggregationClass: 'App\Query\Aggregation\PriorityRangeAggregation'
            $keyMapper: 'Ibexa\Solr\ResultExtractor\AggregationResultExtractor\RangeAggregationKeyMapper\IntRangeAggregationKeyMapper'
```

**Review by papcio122 — CHANGES_REQUESTED** [review]

File: docs/search/extensibility/create_custom_aggregation.md [hunk 4]
```diff
@@ -62,15 +62,15 @@ The example below uses `RangeAggregationVisitor`:
 === "Solr"
 
     ``` yaml
-    App\Query\Aggregation\Solr\PriorityAggregationVisitor:
+    app.search.solr.query.aggregation_visitor.priority_range_aggregation:
         class: Ibexa\Solr\Query\Common\AggregationVisitor\RangeAggregationVisitor
         factory: [ '@Ibexa\Solr\Query\Common\AggregationVisitor\Factory\ContentFieldAggregationVisitorFactory', 'createRangeAggregationVisitor' ]
         arguments:
             $aggregationClass: 'App\Query\Aggregation\PriorityRangeAggregation'
```
Comment (papcio122) [inline docs/search/extensibility/create_custom_aggregation.md:69]: I tested again, but I am still getting some errors, sorry I missed this yesterday
```suggestion
            $aggregationClass: 'App\Query\Aggregation\Solr\PriorityRangeAggregation'
```

File: docs/search/extensibility/create_custom_aggregation.md [hunk 5]
```diff
@@ -106,15 +106,17 @@ and provide it with the aggregation class in the `aggregationClass` parameter.
...
 
-    Tag the service with `ibexa.solr.query.location.aggregation_result_extractor`.
+    Tag the service with `ibexa.search.solr.query.location.aggregation.result.extractor`.
 
     ``` yaml
-    App\Query\Aggregation\Solr\PriorityAggregationResultExtractor:
+    app.search.solr.query.aggregation_result_extractor.priority_range_aggregation:
         class: Ibexa\Solr\ResultExtractor\AggregationResultExtractor\RangeAggregationResultExtractor
         arguments:
             $aggregationClass: 'App\Query\Aggregation\PriorityRangeAggregation'
```
Comment (papcio122) [inline docs/search/extensibility/create_custom_aggregation.md:115]: I tested again, but I am still getting some errors, sorry I missed this yesterday, with these two changes everything works :)
```suggestion
            $aggregationClass: 'App\Query\Aggregation\Solr\PriorityRangeAggregation'
```

**Review by papcio122 — CHANGES_REQUESTED** [review]

**Review by papcio122 — APPROVED** [review]

**Pull request merged**